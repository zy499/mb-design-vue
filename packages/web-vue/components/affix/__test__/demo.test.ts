import demoTest from '../../../scripts/demo-test';

demoTest('affix');
