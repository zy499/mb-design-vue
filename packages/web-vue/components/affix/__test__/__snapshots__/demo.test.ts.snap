// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<affix> demo: render [basic] correctly 1`] = `
"<div>
  <!--v-if-->
  <div class=\\"\\"><button class=\\"mb-btn mb-btn-primary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal\\" type=\\"button\\">
      <!--v-if-->Affix Top
    </button></div>
</div>"
`;

exports[`<affix> demo: render [bottom] correctly 1`] = `
"<div>
  <!--v-if-->
  <div class=\\"\\"><button class=\\"mb-btn mb-btn-primary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal\\" type=\\"button\\">
      <!--v-if-->120px to affix bottom
    </button></div>
</div>"
`;

exports[`<affix> demo: render [container] correctly 1`] = `
"<div style=\\"height: 200px; overflow: auto;\\">
  <div style=\\"height: 400px; background: rgb(204, 204, 204); overflow: hidden;\\">
    <div style=\\"margin: 40px;\\">
      <!--v-if-->
      <div class=\\"\\"><button class=\\"mb-btn mb-btn-primary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal\\" type=\\"button\\">
          <!--v-if-->Affix in scrolling container
        </button></div>
    </div>
  </div>
</div>"
`;

exports[`<affix> demo: render [fix-change] correctly 1`] = `
"<div>
  <!--v-if-->
  <div class=\\"\\"><button class=\\"mb-btn mb-btn-primary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal\\" type=\\"button\\">
      <!--v-if-->80px to affix bottom
    </button></div>
</div>"
`;

exports[`<affix> demo: render [top] correctly 1`] = `
"<div>
  <!--v-if-->
  <div class=\\"\\"><button class=\\"mb-btn mb-btn-primary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal\\" type=\\"button\\">
      <!--v-if-->80px to affix top
    </button></div>
</div>"
`;
